### ISSUE TITLE
DRAIN color change

### ISSUE THREAD

**Issue by jlauro** [haproxy/haproxy#53]
## Output of `haproxy -vv` and `uname -a`
# haproxy -vv
```
HA-Proxy version 1.9.3 2019/01/29 - https://haproxy.org/
Build options :
  TARGET  = linux2628
  CPU     = generic
  CC      = gcc
  CFLAGS  = -O2 -g -fno-strict-aliasing -Wdeclaration-after-statement -fwrapv -Wno-unused-label -Wno-sign-compare -Wno-unused-parameter -Wno-old-style-declaration -Wno-ignored-qualifiers -Wno-clobbered -Wno-missing-field-initializers -Wtype-limits
  OPTIONS = USE_LINUX_TPROXY=1 USE_ZLIB=1 USE_REGPARM=1 USE_OPENSSL=1 USE_PCRE=1

Default settings :
  maxconn = 2000, bufsize = 16384, maxrewrite = 1024, maxpollevents = 200

Built with OpenSSL version : OpenSSL 1.0.2k-fips  26 Jan 2017
Running on OpenSSL version : OpenSSL 1.0.2k-fips  26 Jan 2017
OpenSSL library supports TLS extensions : yes
OpenSSL library supports SNI : yes
OpenSSL library supports : SSLv3 TLSv1.0 TLSv1.1 TLSv1.2
Built with transparent proxy support using: IP_TRANSPARENT IPV6_TRANSPARENT IP_FREEBIND
Built with zlib version : 1.2.7
Running on zlib version : 1.2.7
Compression algorithms supported : identity("identity"), deflate("deflate"), raw-deflate("deflate"), gzip("gzip")
Built with PCRE version : 8.32 2012-11-30
Running on PCRE version : 8.32 2012-11-30
PCRE library supports JIT : no (USE_PCRE_JIT not set)
Encrypted password support via crypt(3): yes
Built with multi-threading support.

Available polling systems :
      epoll : pref=300,  test result OK
       poll : pref=200,  test result OK
     select : pref=150,  test result OK
Total: 3 (3 usable), will use epoll.

Available multiplexer protocols :
(protocols marked as <default> cannot be specified using 'proto' keyword)
              h2 : mode=HTX        side=FE|BE
              h2 : mode=HTTP       side=FE
       <default> : mode=HTX        side=FE|BE
       <default> : mode=TCP|HTTP   side=FE|BE

Available filters :
        [SPOE] spoe
        [COMP] compression
        [CACHE] cache
        [TRACE] trace

[ haproxyce: ~ @ Mon Mar 04 17:38:08 ] 
```
# uname -a
```
Linux haproxyce.ces.cvnt.net 3.10.0-693.5.2.el7.x86_64 #1 SMP Fri Oct 20 20:32:50 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
```

## What should haproxy do differently? Which functionality do you think we should add?

Status color should change when backend is set to DRAIN.

When a backend is put in DRAIN mode the color is not changed and remains green making it non obvious at a glance which backends are in drain mode.  Previously the web status line switched to blue.  I think yellow (as in active UP, going down) might be more appropriate, or switching back to blue, or some other color besides the same shade of green as an active host.  Green is not appropriate as it will not take new sessions when cookies are required.

## What are you trying to do?

Make it obvious at a glance which hosts are active and taking new connections and which ones are not.

**Comment by wtarreau** [comment]
I think it is a bug. We've broken it many times in the past. The last time it was fixed was in 1.6-dev2, but occasionally we manage to lose the drain info when displaying colors, I don't know why. I agree with you that the blue is important to have. I'm pretty sure I've met this issue on my build farm already because I remember having had to read the weight column to figure which nodes were working.

**Comment by wtarreau** [comment]
now applied everywhere, closing.
